src/main/business/UserManager/UserController.js

```diff
@@ -93,10 +93,9 @@ class UserController {
 
     /**
      * Returns all users in the system.
-     * @param session session requesting, must be an Admin.
      */
-    async getAllUsers(session){
-        await this.verifyType(session, USER_TYPES.ADMIN);
+    async getAllUsers(getAllUsersData){
```

Comment:
why not receive `callingUser`?

Reply:
Inside the object there is the callingUser